test/channel_tests.cpp

```diff
@@ -198,6 +199,24 @@ BOOST_FIXTURE_TEST_SUITE(int_channel, channel_test_fixture_int_1)
         BOOST_REQUIRE_EQUAL(42, result);
     }
 
+    BOOST_AUTO_TEST_CASE(move_only_type_channel) {
```

Comment:
There was already a set of move only tests in channel_functor_tests.cpp:151 ff,
so this is not really necessary, or?